cmd/completion/bash_completion.go

```diff
@@ -0,0 +1,354 @@
+package completion
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var activeHelpMarker = "_activeHelp_ "
+
+func genBashCompletion(w io.Writer, appName string, includeDesc bool) error {
+	buf := new(bytes.Buffer)
+	genBashComp(buf, appName, includeDesc)
+	_, err := buf.WriteTo(w)
+	return err
+}
+
+func genBashComp(buf io.StringWriter, appName string, includeDesc bool) {
+	compCmd := cobra.ShellCompRequestCmd
+	if !includeDesc {
```

Comment:
What is the point of this parameter if we always set it to false to avoid processing descriptions ?

Reply:
This file is a copy from the original cobra code: https://github.com/spf13/cobra/blob/main/bash_completions.go. I didn't refractor it in purpose, because when there is a new version, we can almost reintroduce it directly without additional effort.